src/Common/System/LocalAppContextSwitches.Common.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.

```

Comment:
Please update the license headers of all C# files that you bring in to the new 2-line format that we follow in runtime.

Reply:
I think there's a roslyn analyzer that can do that for us, right?

Reply:
Yes. This might be enforced via .editorconfig or a ruleset file.

Reply:
I'll do that separately. I noticed System.Buffers has some old license headers and those can be updated using the analyzer.